runtime/compiler/env/j9method.h

```diff
@@ -182,6 +193,11 @@ class TR_J9Method : public TR_J9MethodBase
 public:
    TR_J9Method(TR_FrontEnd *trvm, TR_Memory *, J9Class * aClazz, uintptr_t cpIndex);
    TR_J9Method(TR_FrontEnd *trvm, TR_Memory *, TR_OpaqueMethodBlock * aMethod);
+
+protected:
+   // To be used by JITServer.
+   // Warning: some initialization must be done manually after calling this constructor
+   TR_J9Method();
```

Comment:
This is protected by `#ifdef JITSERVER_SUPPORT` in the cpp file, so it needs to be protected here as well.